SU-SWS/sulgryphon-nextjs
# SUL23-40 | @jdwjdwjdw | Add styles for News nodes, lists, and cards

# READY FOR REVIEW

# Summary
- [SUL23-40](https://stanfordits.atlassian.net/browse/SUL23-40): FE-DEV | Build "News" content display in NextJS
- Add styles for News nodes, lists, and cards
- Add social icon components for Facebook, Twitter, and linkedin 
- Add additional `footer` field to Card component

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Build out some News pages
2. Confirm the individual news node pages match D9
3. Build default News lists and card grid News list
4. Confirm they match D9

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)


<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/24"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
